sdk/core/core-http/lib/util/constants.ts

```diff
@@ -94,5 +94,59 @@ export const Constants = {
      * @type {string}
      */
     USER_AGENT: "User-Agent"
+  },
```

Comment:
Now that the service specific error handling has moved to the service library, shouldnt the below constants that were added be moved as well?

Reply:
Since the serialization involved in error handling uses both XML data marker constants as well as ODATA, it seemed fine to have it exposed from core-http.
But since they aren't used anywhere in specific in core-http, moved them